app/models/role.rb

```diff
@@ -289,8 +289,10 @@ def allowed_actions
   end
 
   def check_deletable
-    errors.add(:base, _("Cannot delete built-in role")) if builtin?
-    errors.empty?
+    if builtin?
+      errors.add(:base, _("Cannot delete built-in role")) if builtin?
```

Comment:
no need for another if condition?

Reply:
I'd say no, according to the tests and the naming of the method. ;)

Reply:
but you have the same if a line above?

Reply:
fixed.